### PR TITLE
skip_file macro replaced with skip macro.

### PR DESCRIPTION
**Pull request by akzhan**
Follow https://github.com/crystal-lang/crystal/pull/4675#issuecomment-313411432

### CODE REVIEW

File: src/crystal/system/unix/arc4random.cr [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% skip_file unless flag?(:openbsd) %}
+{% skip() unless flag?(:openbsd) %}
```
Comment (RX14) [inline src/crystal/system/unix/arc4random.cr]: Does it require the empty call args?

Comment (akzhan) [inline src/crystal/system/unix/arc4random.cr]: I'm unsure but `skip` macro is tested and documented with `()`.

Comment (akzhan) [inline src/crystal/system/unix/arc4random.cr]: `()` are required due to

```
  1) Semantic: macro skip macro directive skips expanding the rest of the current file

       Error in line 6: expanding macro

       in line 6: undefined macro variable 'skip'
```

**Comment by ysbaddaden** [comment]
Why?

**Comment by akzhan** [comment]
@ysbaddaden Current codebase fail on ` bin/crystal doc`, see  discussion started with https://github.com/crystal-lang/crystal/pull/4675#issuecomment-313386287

**Comment by straight-shoota** [comment]
Wouldn't it make more sense to rename everything to `skip_file`? That's more descriptive of what is does.

**Comment by ysbaddaden** [comment]
Thanks. Always link or copy-paste explanations. We do not necessarily notice buried comments.

Keep `skip_file()` with forced parens, so it's obvious to the compiler that it's a method call. This macro name is more descriptive than a clueless `skip`.

**Comment by mverzilli** [comment]
`skip_file` was the original name, then we decided to change it to `skip` (`skip_file` only existed for a couple of days in master). I guess in the meantime we merged the code @akzhan is trying to fix now. 

I see your point that `skip_file` is more descriptive than `skip`, and given it's not going to be a very commonly used feature, more clarity is well worth the extra keystrokes. 

So I'd suggest that we closed this PR and opened another one renaming `skip` to `skip_file`.

**Comment by akzhan** [comment]
It's time to prefer `skip_file`.

**Pull request closed without merging**